importer/RuRu.py

```diff
@@ -0,0 +1,107 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class RuRu(Monument):
+
+    def set_adm_location(self):
+        """
+        Set the administrative location.
+
+        Preferably try to resolve the 'district' (район/округ,
+        the field contains different types so that's why we're
+        matching it against the whole big list of administrative
+        divisions of different kinds).
+        If that doesn't work, use the higher level субъект.
+        """
+        level_three = self.data_files["administrative"]
+        level_two = self.data_files["subyekty"]
+        district_match = [x for x in level_three if x[
```

Comment:
```python
district_match = [x for x in level_three
                  if x["itemLabel"] == self.district]
```
to avoid line breaking in the middle of the `if`